.servicebot/audit.log

```diff
@@ -0,0 +1 @@
+confluentinc/go-ps1 - file updated with different contents:  path=/Users/abrown/confluent/temp/go-ps1/.pre-commit-config.yaml
```

Comment:
Not a huge fan of adding yet another file/directory to the top-level... what does this audit log file accomplish that git history doesn't?

Reply:
it was an accidental addition.